From patchwork Wed May  4 21:15:30 2016
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [U-Boot,2/6] Revert "sunxi: Reserve ATF memory space on A64"
From: Andre Przywara <andre.przywara@arm.com>
X-Patchwork-Id: 618676
Message-Id: <1462396534-32390-3-git-send-email-andre.przywara@arm.com>
To: Hans de Goede <hdegoede@redhat.com>,
	Ian Campbell <ijc@hellion.org.uk>
Cc: Andre Przywara <andre.przywara@arm.com>, u-boot@lists.denx.de
Date: Wed,  4 May 2016 22:15:30 +0100

The ARM Trusted Firmware (ATF) code now lives in SRAM on the Pine64/A64,
so we can claim the whole of DRAM for OS use.

This reverts commit 3ffe39ed2b66af71c7271d0cef2a248b5bf7dfdb.

Signed-off-by: Andre Przywara <andre.przywara@arm.com>
---
 board/sunxi/board.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/board/sunxi/board.c b/board/sunxi/board.c
index 3cf3614..ccf4129 100644
--- a/board/sunxi/board.c
+++ b/board/sunxi/board.c
@@ -133,15 +133,6 @@ int dram_init(void)
 	return 0;
 }
 
-#ifdef CONFIG_MACH_SUN50I
-void dram_init_banksize(void)
-{
-	/* We need to reserve the first 16MB of RAM for ATF */
-	gd->bd->bi_dram[0].start = CONFIG_SYS_SDRAM_BASE + (16 * 1024 * 1024);
-	gd->bd->bi_dram[0].size = get_effective_memsize() - (16 * 1024 * 1024);
-}
-#endif
-
 #if defined(CONFIG_NAND_SUNXI) && defined(CONFIG_SPL_BUILD)
 static void nand_pinmux_setup(void)
 {
